src/main/java/edu/ucsb/cs156/happiercows/entities/CommonStats.java

```diff
@@ -0,0 +1,29 @@
+package edu.ucsb.cs156.happiercows.entities;
+
+import java.time.LocalDateTime;
+import java.util.Date;
+
+import javax.persistence.*;
+
+import org.hibernate.annotations.CreationTimestamp;
+
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+@Entity(name = "commonstats")
+
+public class CommonStats {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private long id;
+    private long commonsId;
+    private int numCows;
+    private double avgHealth;
+    private LocalDateTime timestamp;
```

Comment:
```suggestion
    @CreatedDate
    private LocalDateTime timestamp;
```

If we add this, with import:
```java
import org.springframework.data.annotation.CreatedDate;
```

then the field will be automatically populated with the correct value when you save a new record in the database.